README.md

```diff
@@ -52,9 +54,20 @@ Write tests in the form of `<expected output or action>` from `<input>` / when `
 Add to these tests:
 
 1. Write minimum and maximum to the PDF from a csv containing positive and negative readings
-1. Write "Invalid input" to the PDF when the csv doesn't contain expected data
-1. _enter a test
-1. _enter a test
+2. Write "Invalid input" to the PDF when the csv doesn't contain expected data
+3. Write "Threshold Crossed" to the PDF whenever exceeds the maximum value or recedes the minimum value
+4. Write the Count of breaches in the pdf with the time stamps whenever breach happens.
+5. Write the Count of breaches after the 7 days count
+6. Show the trend of the battery after the 7 days battery telemetrics
+7. Write "No CSV file found" when the proper file format not present or no file present.
+8. Write "Server not acessible" when the server for storing pdf is not reachable
```

Comment:
Line Item 5.....14 contains "Write *************" ...question is "to where" ? are we missing any hidden dependency!